src/modules/0.0.19/iasql_functions/index.ts

```diff
@@ -1,10 +1,15 @@
 /* THIS MODULE IS A SPECIAL SNOWFLAKE. DON'T LOOK AT IT FOR HOW TO WRITE A REAL MODULE */
+import { readFileSync } from 'fs';
 import { ModuleBase } from '../../interfaces';
 import { IasqlOperationType } from './entity';
 
 class IasqlFunctions extends ModuleBase {
   constructor() {
     super();
+    this.sql = {
+      afterInstall: readFileSync(`${__dirname}/sql/create_fns.sql`, 'utf8'),
+      beforeUninstall: readFileSync(`${__dirname}/sql/drop_fns.sql`, 'utf8')
```

Comment:
not a fan of this but `this.dirname` is available after `init()` and we need this first

Reply:
There's a better solution here.

The same module that's used to figure out the `dirname` property could be used here. The `sql` object could be defined statically on the module just with a relative path string for the `sql` files, and then the `init` logic could replace them with the contents of the files.

Reply:
oh, that definitely will work